collector/container.go

```diff
@@ -95,38 +96,38 @@ func newContainerMetricsCollector() (Collector, error) {
 		),
 		BytesReceived: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "network_receive_bytes_total"),
-			"Bytes Received on Interface",
-			[]string{"container_id", "interface"},
+			"Bytes Received on HNS endpoint",
+			[]string{"shared_containers", "endpoint_id"},
 			nil,
 		),
 		BytesSent: prometheus.NewDesc(
 			prometheus.BuildFQName(Namespace, subsystem, "network_transmit_bytes_total"),
-			"Bytes Sent on Interface",
-			[]string{"container_id", "interface"},
+			"Bytes Sent on HNS endpoint",
+			[]string{"shared_containers", "endpoint_id"},
```

Comment:
@jsturtevant Network stats are going to be per HNS endpoint. Each endpoint can be associated with zero or more container ids. 

We can associate each endpoint network metric with the label 
- `shared_containers`
- `container_ids`
- or something else

What would you recommend? It made sense to name the label `shared_containers` because that is the name the the underlying HNS component uses.

Reply:
This should stay the same otherwise downstream consumers will break.  Since we are putting the endpoint ID into the metric, it will be up to tooling to stitched it together properly.

This is the same thing that happens today on the docker v1 schema. If you were to look at a dockershim implementation each container would get the same value but would have the shared "interface". The tooling putting it together would need to know about that the containers share the interface and should group the together based on interface.

Reply:
Got it, will make the change so we don't introduce breaking changes.

Reply:
I am resolving this conversation. I have since reverted to the initial metric naming and labelling to avoid making a breaking change.